lms/templates/discovery/course_card.underscore

```diff
@@ -12,7 +12,12 @@
                 <span class="course-code"><%= content.number %></span>
                 <span class="course-title"><%= content.display_name %></span>
             </h2>
-            <div class="course-date" aria-hidden="true"><%= interpolate(gettext("Starts: %s"), [start]) %></div>
```

Comment:
@dsego gettext doesn't work in the same line as interpolate in underscore templates. Please be sure that you're properly testing your features for i18n. Further, please make sure to give formatting variables proper names when used in translator comments.